core/php7.1Action/CHANGELOG.md

```diff
@@ -0,0 +1,44 @@
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+## 1.0.1
+
+- Change: Update PHP to 7.1.8
+- Change: Update [guzzlehttp/guzzle](https://packagist.org/packages/guzzlehttp/guzzle) to 6.3.3
+- Change: Update [ramsey/uuid](https://packagist.org/packages/ramsey/uuid) to 3.7.3
+
+## 1.0.0
+Initial release
+
+- Added: PHP: 7.1.9.
```

Comment:
was this 7.1.9?

Reply:
> was this 7.1.9?

I believe so. At least, that's what `openwhisk/action-php-v7.1:latest` from Docker Hub has.